.github/scripts/deploy_pages.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e -x
+
+tar -xf html.tar.gz
+rm -f html.tar.gz
+git checkout gh-pages
+find * -maxdepth 0 -not -path "html*" -exec rm -rf {} \;
+cp -r html/* .
+rm -rf html
+
+touch .nojekyll # https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/
+
+git config --local user.email "Valentina.Kats@intel.com"
+git config --local user.name "Valentina Kats"
+git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}
```

Comment:
Is it really needed? It doesn't have it: https://github.com/oneapi-src/oneAPI-spec/blob/main/.github/workflows/ci.yml#L48

Reply:
CI script updated, please check